Automattic/jetpack
# Sync: move the IDC initialization to the Sync package

The IDC feature should always be enabled if the Sync package is used. Let's initialize the IDC feature in the Sync package so that consumer plugins do not need to initialize it separately.

#### Changes proposed in this Pull Request:
* Initialize the IDC package in the Sync `Main::configure` method.

#### Jetpack product discussion
* Discussion here: p1623336642470300-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
Test by causing an identity crisis and verifying that the IDC UI is displayed.

1. Install, activate, and connect this branch of Jetpack.
2. Cause an IDC. For example, you can do something like this:
   a. Add this code snippet:
      ```
     add_filter( 'jetpack_sync_site_url', function ( $value ) {
	    if( defined('XMLRPC_REQUEST') && XMLRPC_REQUEST ) {
	    	return $value;
	    }
        	return 'http://example.org/site';
    } );
    ```
    b. Add a new post or edit an existing post.
3. Navigate to the Jetpack dashboard. The IDC UI should be displayed.
